Example/PrimerSDK/MerchantCheckoutViewController.swift

```diff
@@ -150,7 +150,7 @@ extension MerchantCheckoutViewController: PrimerDelegate {
         request.httpMethod = "POST"
         request.addValue("application/json", forHTTPHeaderField: "Content-Type")
         
-        let body = CreateClientTokenRequest(customerId: "customer123", customerCountryCode: nil)
+        let body = CreateClientTokenRequest(customerId: "customer123", customerCountryCode: nil, staging: true)
```

Comment:
Same as above. I don't see it used anywhere.

Reply:
It's just for the example app's merchant integration to know we're targeting staging.